client/components/token-field/index.jsx

```diff
@@ -90,21 +101,30 @@ var TokenField = React.createClass( {
 
 	render: function() {
 		var classes = classNames( 'token-field', {
-			'is-active': this.state.isActive
+			'is-active': this.state.isActive,
+			'is-disabled': this.props.disabled
 		} );
 
+		var tokenFieldProps = {
+			ref: 'main',
+			className: classes,
+			tabIndex: '-1'
```

Comment:
When I search, I'm seeing articles that suggest we should leave `tabIndex: -1` here.

`If given a value of "-1", the element can't be tabbed to but focus can be given to the element programmatically`